src/gui/guiHyperText.h

```diff
@@ -177,6 +179,8 @@ class TextDrawer
 	inline s32 getHeight() { return m_height; };
 	void draw(const core::rect<s32> &clip_rect,
 			const core::position2d<s32> &dest_offset);
+	void drawBackgroundImage(video::IVideoDriver *driver, const ParsedText &m_text, const core::rect<s32> &clip_rect);
```

Comment:
Why passes `ParserdText` as argument in this method? `m_text` can be obtained from instance.